examples/wireless_wpa3_owe.yml

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  vars:
+    network_connections:
+      - name: wlan0
+        type: wireless
+        wireless:
+          ssid: "My WPA3-Personal Network"
+          key_mgmt: "owe"
+          password: "p@55w0rD"
```

Comment:
Like the `wireless_wpa_psk.yml`. please suggest user to use the vault.

Reply:
> > not sure why CI "centos-8/ansible-2.8 (staging)" failed, I tested correctly locally with "CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2"(image downloaded from https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2)
> 
> `centos-8/ansible-2.8` is using CentOS 8.2 - it fails in ethtool_coalesce - `centos-8/ansible-2.8 (staging)` is using CentOS 8.3 - it fails in what looks like the code from this PR - so I suggest you focus on using CentOS 8.3 - can you test with that locally?

It failed locally, I will investigate it